source/agora/test/Base.d

```diff
@@ -228,6 +228,9 @@ public struct Serializer
 /// A different default serializer from `LocalRest` for `RemoteAPI`
 public alias RemoteAPI (APIType) = geod24.LocalRest.RemoteAPI!(APIType, Serializer);
 
+/// The number of validators in the genesis block
+public static immutable uint ValidateCountInGenesis = 2;
```

Comment:
Where does this come from ? And why 2 instead of 1 ?

Reply:
It was recorded as 2 in Issue 822, and in the article above you only said it was a fixed value, but you didn't specify the exact value.

Reply:
It was mentioned as a possibility, but it's up  to the person implementing the feature what makes sense and to explain why. I'll answer on the issue itself.